### PR TITLE
feat: #1189 Update successful and warning notification when adding or deleting application admin

### PR DESCRIPTION
**Pull request by J0taFerreira**
- The error message is being dismissed when the user clicks something else
- changed the error icon

### CODE REVIEW

**Comment by MCatherine1994** [comment]
Hi Jota, let's check sonar cloud first, there are some minor fixes we can do

**Review by NickSaglioni — APPROVED** [review]
I don't like calling `resetNotification()` at so many different places but at the same time I don't know a better alternative for that, so I think it is ok for now. Thanks Jota.

**Comment by J0taFerreira** [comment]
> I don't like calling `resetNotification()` at so many different places but at the same time I don't know a better alternative for that, so I think it is ok for now. Thanks Jota.

Me neither, I tried doing it in some reusable way but did not manage to find a suitable way

File: frontend/src/store/NotificationState.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ export const setNotificationMsg = (
     msg: string = '',
     fullMsg: string = ''
 ) => {
+    clearNotification(severity)
```
[the remark this answers is not in the document]
Comment (J0taFerreira) [reply frontend/src/store/NotificationState.ts]: thanks, the way you suggested works!

File: frontend/src/components/grantaccess/UserIdentityCard.vue [hunk 2]
```diff
@@ -20,34 +20,76 @@ const props = defineProps<{
...
             />
             <p>Verified user information</p>
         </template>
         <template #content>
-            <div class="col" style="margin-left: 2rem">
-                <label class="row">Username</label>
-                <span class="row" id="userId">{{
-                    props.userIdentity.userId
-                }}</span>
+            <div
```
[the remark this answers is not in the document]
Comment (J0taFerreira) [reply frontend/src/components/grantaccess/UserIdentityCard.vue:31]: That's the way prettier formatted it, but I can try to make closer to the way it was

**Review by MCatherine1994 — APPROVED** [review]

**Pull request merged**